### PR TITLE
doc: Add 1.5.1 to master

### PR DESCRIPTION
**Pull request by umapraseeda**
Update master with 1.5.1 files
Updates known issues with 1.5.1 tag

Signed-off-by: Uma Praseeda <uma.praseeda@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/known_issues.rst [hunk 1]
```diff
@@ -349,7 +447,7 @@ DESK-978: Directed advertising issues with SoftDevice Link Layer
```
Comment (ru-fu) [inline doc/nrf/known_issues.rst:447]: I just spotted that the note right above isn't indented correctly, and thus shows up when it should be filtered out. No need to change for the other PR imo, but you could fix it here for master.

**Review by ru-fu — COMMENTED** [review]
Looks good, just one comment about the known issues.

File: doc/nrf/known_issues.rst (hunk 1 above)
Comment (umapraseeda) [reply doc/nrf/known_issues.rst:447]: Fixed

**Review by ru-fu — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]
Once this PR is merged, please merge also the following one: https://github.com/nrfconnect/sdk-nrf/pull/4326

**Review by rlubos — APPROVED** [review]

**Pull request merged**